uniflow-androidx-test/src/main/java/io/uniflow/android/test/SimpleTestObservers.kt

```diff
@@ -34,7 +33,14 @@ class TestViewObserver {
 
     @Deprecated("better use verifySequence")
     fun assertReceived(vararg any: UIData) = verifySequence(*any)
-    fun verifySequence(vararg any: UIData) = assert(this.values == any.toList()) { "Wrong values\nshould have [${any.toList()}]\nbut was [${values}]" }
+    fun verifySequence(vararg testingData: UIData){
+        val testingValues = testingData.toList()
+        values.forEachIndexed { index, uiData ->
+            assert(uiData == testingValues[index]) {
+                "Wrong values at [$index] - Type: ${uiData::class.simpleName}\nShould have [${testingValues[index]}]\n  but was [${uiData}]"
+            }
```

Comment:
Same comments apply as for uniflow-android-test/src/main/java/io/uniflow/android/test/SimpleTestObservers.kt.